src/classes/RD2_VisualizeScheduleController.cls

```diff
@@ -46,6 +46,20 @@ public with sharing class RD2_VisualizeScheduleController {
         } set;
     }
 
+    /***
+    * @description Recurring Donation custom settings
+    */
+    private static Date earliestMatchDate {
```

Comment:
`earliestMatchDate ` -> `startDate` so we are consistent throughout the code base